client/landing/stepper/declarative-flow/internals/steps-repository/blogger-starting-point/index.tsx

```diff
@@ -27,32 +28,35 @@ const StartingPointStep: Step = function StartingPointStep( { navigation } ) {
 	};
 
 	return (
-		<StepContainer
-			stepName="blogger-starting-point"
-			headerImageUrl={ startingPointImageUrl }
-			goBack={ goBack }
-			skipLabelText={ translate( 'Skip to dashboard' ) }
-			goNext={ () => submitIntent( 'skip-to-my-home' ) }
-			skipButtonAlign="top"
-			isHorizontalLayout={ true }
-			formattedHeader={
-				<FormattedHeader
-					id="intent-header"
-					headerText={ headerText }
-					subHeaderText={ subHeaderText }
-					align="left"
-				/>
-			}
-			stepContent={
-				<IntentScreen
-					intents={ intents }
-					intentsAlt={ [] }
-					onSelect={ submitIntent }
-					preventWidows={ preventWidows }
-				/>
-			}
-			recordTracksEvent={ recordTracksEvent }
-		/>
+		<>
+			<DocumentHead title={ translate( 'Blogger Starting Point' ) } />
+			<StepContainer
```

Comment:
Is it possible to make `DocumentHead` inside `StepContainer`?

Reply:
You mean to pass the title as props? Should we update the placement on all steps, such as import https://github.com/Automattic/wp-calypso/blob/52c0c79d55317d5922f5d92a84d8dd784f71e337/client/landing/stepper/declarative-flow/internals/steps-repository/import-light/index.tsx#L125

Reply:
Yes